src/tests/test_util.py

```diff
@@ -130,6 +133,30 @@ def test_get_2of2_selections(self, *args):
 
         self.assertEqual(get_selection(roles), roles[1])
 
+    @patch('builtins.input', return_value=3)
+    @patch('sys.stdout', new=StringIO())
+    def test_get_bad_numeric_selection(self, *args):
+        """ Invalid numeric selection of two roles """
+        roles = [
+            ('idp1', 'arn:aws:iam::224588347132:role/KalturaAdmin'),
+            ('idp2', 'arn:aws:iam::617683844790:role/BoxAdmin'),
+        ]
+
+        with self.assertRaises(InvalidSelection):
+            get_selection(roles)
+
+    @patch('builtins.input', return_value="foo")
+    @patch('sys.stdout', new=StringIO())
+    def test_get_bad_type_selection(self, *args):
+        """ Invalid string selection of two roles """
+        roles = [
+            ('idp1', 'arn:aws:iam::224588347132:role/KalturaAdmin'),
```

Comment:
Same as above.

Reply:
@JonRoma sorry I accidentally deleted you comment above. Not sure why I am able to do that...

I can change the account number, I usually do scramble them, but this account number has been out in the public for more than two years so I'm not sure there is any point in changing it. I would prefer not to rewrite the entire public history to remove the account id here, above, and in six other locations. If you and @edthedev think it is necessary I will do so, but I don't think the risk is worth the effort or disruption it could cause to users of this code base.

Reply:
SDG does not have standards for a `CHANGELOG` since we have so little code that's open to the public. Is there a PEP on the subject?

I would probably not go back to correct this omission for the older releases, but it would be good to document the recent round of improvements and bug fixes.

Reply:
@ddriddle, unless @edthedev thinks you should rewrite the history to remove the account ID, I would be satisfied if it were to become scrambled henceforth. It's not as vital to expunge it as it would be a password, so having it go away in future releases would be good enough for me. Thanks.

Reply:
@JonRoma , this is a good source for CHANGELOG standards: https://keepachangelog.com/en/1.0.0/